src/main/resources/templates/createquestion.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+<h1>Create Question</h1>
+
+<label for="questionType">Select A Question:</label>
+<select id="questionType" name="questionType" required="required" onchange="update(this.value)" value="'TEXT'">
+    <option value="" selected disabled hidden>-- Select a Question --</option>
+    <option value="TEXT">Text Question</option>
+    <option value="NUMBER_CHOICE_LINE">Number Question</option>
+    <option value="MULTIPLE_CHOICE">Multiple Choice Question</option>
+</select><br><br>
+
+<div id="questionContent">
+</div>
+
+
+<script th:inline="javascript">
+    var type = "";
+    let externalHTML = "";
+
+    function update() {
+        type = document.getElementById("questionType").value;
+        const main = document.getElementById('questionContent');
+        if (type == "TEXT") {
+            externalHTML = `
+                <form method="post" action="/textquestion">
+                    <label for="description">Description:</label>
+                    <input type="text" id="description" name="description" required><br><br>
+                    <label for="charLimit">Character Limit:</label>
```

Comment:
There should be some kind of client-side validation for this as users can enter text into them when there should only be numbers.

Reply:
Just added a change for this! Now only character limit, min range and max range can be numbers